config/application.rb

```diff
@@ -19,7 +19,8 @@ class Application < Rails::Application
 
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    #config.time_zone = 'Central Time (US & Canada)'
+    ENV['TZ']='UTC'
```

Comment:
yes and no.  If this wasn't set the entire site would break if ever run on a non utc server.

Although the only place it gets run on a non-utc server is when testing.

I put it here because it is a system requirement that it be run in the UTC timezone, not just a testing requirement.